en/components/grids_templates/sizing.md

```diff
@@ -215,6 +215,11 @@ When columns have set specific `width` in pixels, they stick to that size, unles
 
     <img src="../../images/grid_sizing/columns-150px-extra-area-v2.jpg" style="width: 80%"/>
 
+### Auto
+
+When columns are set to `auto` the behaviour is the same as when using pixels the only difference is the size of the column is not a fixed value like in pixels but rather adjusts to fit the longest word in the column. Empty areas are possible.
```

Comment:
The first sentence appears to be too long  and  giving too much info at once. It would be easier to understand if it is split into smaller sentences.

Reply:
@nikolam001 Can you take a look at the comment from @skrustev

Reply:
Addressing @skrustev's comment.